### PR TITLE
Refactor/plans in definition run e2e

### PR DESCRIPTION
**Pull request by marcambier**
**Issue**

https://github.com/gravitee-io/issues/issues/7693

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/refactor-plans-in-definition-run-e2e/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-wqlpsbjrax.chromatic.com)

### CODE REVIEW

File: gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/ApiConverter.java [hunk 1]
```diff
@@ -66,11 +67,7 @@ public ApiEntity toApiEntity(Api api, PrimaryOwnerEntity primaryOwnerEntity) {
 
         if (api.getDefinition() != null) {
             try {
-                io.gravitee.definition.model.Api apiDefinition = objectMapper.readValue(
-                    api.getDefinition(),
-                    io.gravitee.definition.model.Api.class
-                );
-
+                var apiDefinition = objectMapper.readValue(api.getDefinition(), io.gravitee.definition.model.Api.class);
```
Comment (gcusnieux) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/ApiConverter.java:70]: `var` ?

**Review by gcusnieux — APPROVED** [review]

**Review by ThibaudAV — APPROVED** [review]
Nice work, GG

File: gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/ApiConverter.java (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/ApiConverter.java:70]: :Yann-evil-no-see:

Comment (marcambier) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/ApiConverter.java:70]: :+1:

**Pull request merged**